### ISSUE TITLE
Cable wiring for 2209

### ISSUE THREAD

**Issue by phpbbireland** [Chaoticlab/CNC-Tap-for-Voron#9]
If you connect the connector to either of the EBB 2209 boards with orientation shown, V and G will be reversed possibly damaging the sensor board. 
![connector](https://github.com/Chaoticlab/CNC-Tap-for-Voron/assets/219282/4342cb8b-103e-4874-ab11-33c9e5d713ee)

**Comment by phpbbireland** [comment]
Correct wiring... 
![20240310_090455](https://github.com/Chaoticlab/CNC-Tap-for-Voron/assets/219282/13c23548-08ad-43f5-bf1b-78e4bbaf3614)
Can you confirm if this is correct?

**Comment by Mugga6315** [comment]
I don't think that this is the correct orientation. I did it that way, light from tap is working and in klipper I get reading. But its always triggered when board klipper is started. Even when I switch in config. Seems that maybe G / S have to be switched. But I wanted to check first

**Comment by Mugga6315** [comment]
Can confirm, that for my setup this wiring on your picture is working.

**Comment by hiddengerbil** [comment]
I have a premade cable, which fits that probe socket, and it has a tap plug and a two pin Filament run-out detector. Is the filament detector still possible with your wiring example?

**Comment by JohnTasto** [comment]
The connector diagram on the bottom looks like a copypasta error lifted from the BTT EBB docs. This needs to be fixed.

**Comment by phpbbireland** [comment]
This has not been corrected in the PDF...

The connector on top of the image in the PDF is correct, it shows the 5 Volts (Red Cable) on the right.

However the cable on bottom of the image has the 5 Volts on the left, so naturally you would flip it to ensure you connect the 5Volts (red wires)... but then the connectors Key will prevent inserting it...

At this point you will check the wiring and correct but on a bad hair day you might just plug the cable using the key... something may go bang!
